Feature/match request (#9)
adzerk/audience-ios-objc-sdk: docs/README_BODY.md

### Send a track event

> A tracking event will log an user action in Velocidi's CDP.

Reviewer: ```suggestion
A tracking event will log a user action in Velocidi's CDP.
```

```

If you had any problem with importing the Objective-C class into Swift, please take a look at Apple's guide on [Importing Objective-C into Swift](https://developer.apple.com/documentation/swift/imported_c_and_objective-c_apis/importing_objective-c_into_swift).

### Make a match

> Match requests are used to link multiple identifiers in Velocidi's CDP. This way, any action made with any of the identifiers, across multiple channels (Browser, Mobile App, etc..), can be associated to the same user.

Reviewer: ```suggestion
Match requests are used to link multiple identifiers in Velocidi's CDP. This way, any action made with any of the identifiers, across multiple channels (Browser, Mobile App, ...), can be associated to the same user.
```

> In VelocidiSDK, a match request will link the user's Advertising Identifier with other provided identifiers (like an internal ID). A typical use case for this is, for instance, during the login action, to associate the user's ID with Apple's Advertising Identifier (identifier used in all the tracking event requests).

Reviewer: What do you think?

```suggestion
With this SDK, a match request will link the user's Advertising Identifier with other provided identifiers (like an internal ID). A typical use case for this is, for instance, during the login action, to associate the user's ID with Apple's Advertising Identifier (identifier used in all the tracking event requests).
```

Author: I'd say this is the same thing

Reviewer: It is, but to me it seems less confusing, but I don't have strong feelings. If you prefer to keep as is, then please fix formatting:

```suggestion
In Velocidi SDK, a match request will link the user's Advertising Identifier with other provided identifiers (like an internal ID). A typical use case for this is, for instance, during the login action, to associate the user's ID with Apple's Advertising Identifier (identifier used in all the tracking event requests).
```

Author: But VelocidiSDK is the name of the framework, the formatting is not wrong 😅

Reviewer: 🙈

__Swift__
```swift
@IBAction func sendMatchEvent(_ sender: Any) {
    let userId1 = VSDKUserId(userId: "bar", "fooType")
    let userId2 = VSDKUserId(userId: "baz", "fooType")
    let idsArray = NSMutableArray(array: [userId1, userId2])
      
    VSDKVelocidi.sharedInstance().match("1234-providerId-56789", userIds: idsArray, onSuccess:{ (response: URLResponse, responseObject: Any) in
        print("Success! Response: \(response)")
    }, onFailure:{(error: Error) in
        print("Failed! Error: \(error.localizedDescription)")
    })
}
```

__Objective-C__

> ```objc

Reviewer: ```suggestion
```objectivec
```

Author: it's equivalent https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L3381

Author: During an offline conversation, we agreed that we should use `objectivec` because the body will be used in other projects that use [PrismJS](https://github.com/PrismJS/prism/blob/master/components.json) to render this."

Changed in 21e1335
